app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -80,8 +80,8 @@ def log_clone_options(clone_options)
     _log.info("Destination VM Name:        [#{clone_options[:name]}]")
     _log.info("Destination Host:           [#{clone_options[:host].name} (#{clone_options[:host].ems_ref})]")
     _log.info("Destination Datastore:      [#{clone_options[:datastore].name} (#{clone_options[:datastore].ems_ref})]")
-    _log.info("Destination Folder:         [#{clone_options[:folder].name}] (#{clone_options[:folder].ems_ref})")
-    _log.info("Destination Resource Pool:  [#{clone_options[:pool].name} (#{clone_options[:pool].ems_ref})]")
+    _log.info("Destination Folder:         [#{clone_options[:folder].name}] (#{clone_options[:folder].ems_ref})") if clone_options[:folder]
+    _log.info("Destination Resource Pool:  [#{clone_options[:pool].name} (#{clone_options[:pool].ems_ref})]") if clone_options[:pool]
```

Comment:
I think it makes sense to close this PR since we need both of these by the time we submit the call to VMware.  The real bug is failing to find a default folder if one isn't provided.